examples/sveltekit-live-avatars/README.md

```diff
@@ -0,0 +1,35 @@
+<p align="center">
+  <a href="https://liveblocks.io">
+    <img src="https://liveblocks.io/icon-192x192.png" height="96">
+  </a>
+</p>
+
+# [Liveblocks](https://liveblocks.io) × [SvelteKit](https://kit.svelte.dev/).
+
+This repo shows how to implement live avatars with Liveblocks and [SvelteKit](https://kit.svelte.dev/). [Demo](https://sveltekit-live-avatars.vercel.app/)
```

Comment:
I'm wondering if we should link to liveblocks.io avatars example since it's the exact same output (even if they are not built with Svelte) 🤔